packages/server/src/fhir/operations/export.test.ts

```diff
@@ -106,37 +104,40 @@ describe('System export', () => {
     const accessToken = await initTestAuth();
     expect(accessToken).toBeDefined();
 
-    const res1 = await request(app)
-      .post(`/fhir/R4/Patient`)
-      .set('Authorization', 'Bearer ' + accessToken)
-      .set('Content-Type', 'application/fhir+json')
-      .send({
-        resourceType: 'Patient',
-        name: [{ given: ['Alice'], family: 'Smith' }],
-        address: [{ use: 'home', line: ['123 Main St'], city: 'Anywhere', state: 'CA', postalCode: '90210' }],
-        telecom: [
-          { system: 'phone', value: '555-555-5555' },
-          { system: 'email', value: 'alice@example.com' },
-        ],
-      });
-    expect(res1.status).toBe(201);
-
     // Create observation
-    const res2 = await request(app)
-      .post(`/fhir/R4/Observation`)
-      .set('Authorization', 'Bearer ' + accessToken)
-      .set('Content-Type', 'application/fhir+json')
-      .send({
-        resourceType: 'Observation',
-        status: 'final',
-        code: { text: 'test' },
-        subject: { reference: `Patient/${res1.body.id}` },
-      });
-    expect(res2.status).toBe(201);
-
+    let obs1;
+    let updatedDate = new Date();
     await waitFor(async () => {
+      const res1 = await request(app)
```

Comment:
cleanup to move all app requests inside `waitFor`

Reply:
I don't think I follow this - can you help me understand why we use `waitFor` here?

Reply:
Good catch `waitFor` was not needed. 
I ended up the test cases since these flaky tests were not providing any added test coverage.